### ISSUE TITLE
new password and old password are seen on the logs

### ISSUE THREAD

**Issue by Thus0** [wosc/webpasswd#1]
in the logs, I can see the passwords in clear text as they are provided on the command line, which is a security issue for me 

```
juin 01 09:22:23 ctf sudo[106334]:       tz : TTY=pts/5 ; PWD=/home/thus0/usr/venv/webpasswd/lib/python3.11/site-packages/ws/webpasswd ; USER=root ; COMMAND=/home/thus0/usr/venv/webpasswd/bin/webpasswd-change foobar pass123 pass321
```
How could web.py pass the arguments securely to update.py... perhaps through a temporary file ? We cannot pass via environment variables as sudo prohibit it.

**Comment by Thus0** [comment]
I can provide a PR to pass the password in a temporary file instead of args in the command line, so the password are not visible in the logs.

https://github.com/Thus0/webpasswd/tree/2.2.1-dev

**Comment by wosc** [comment]
You're absolutely right, this is not a very secure implementation, that's why I put a big warning on it. ;)

I've always been to lazy to do this properly, and it did not matter for my small usecase, but I've now [read up on the topic a little](https://www.netmeister.org/blog/passing-passwords.html) and AFAIU passing sensitive data via file descriptors (e.g. stdin) is the best practise here. Which I've implemented in c8098fc and released as 2.2.0. Thanks for nudging me to take a look at this. :)
